### DISCUSSION TITLE
Problem with New-PSSession - content type is absent or invalid

### DISCUSSION THREAD

**Discussion by dominikjeske** [PowerShell/PowerShell#21412] [General]
I have weird problem with New-PSSession command. When I try to connect to server with

`New-PSSession -ComputerName "xxx"`

I have response

`The WinRM client cannot process the request. It cannot determine the content type of the HTTP response from the destination computer. The content type is absent or invalid. For more information, see the about_Remote_Troubleshooting Help topic.`

But other users can connect to machine. The funny thing is that the same error occurs when logged on this server (I have an error but other person can execute) so it looks like some permission. I tried 

`Set-PSSessionConfiguration -Name "PowerShell.7" -ShowSecurityDescriptorUI`

and I can see Administrators can do everything and I'm and administrator on that machine. I also tried tips from this site https://support.entuity.com/hc/en-us/articles/360019058197-How-do-I-configure-and-troubleshoot-WinRM but I still have same error. Any ideas?

**Comment by jborean93** [comment]
Do you potentially have a proxy that is being used with the connection. The error about an invalid content type would indicate that something is responding to the WinRM HTTP request but it's replying with something that is not the expected WinRM response. I would try and capture the network traffic to see what the raw responses are to debug further.

**Reply by dominikjeske** [reply]
@jborean93 This is one thing I was thinking about but what about case when two persons are logging to this server by remote desktop and invoke connection to this server (so we are on same machine) and one has this error and socond is not. This thing is driving me nuts :)

**Reply by jborean93** [reply]
That shouldn't matter, a WinRM logon is completely separate from the RDP session, it's started as a network logon. I can't rule it out but ultimately you should enable the WinRM even log tracing to see what the raw responses are or use Wireshark after disabling the encryption settings to see the raw responses.
